File: app/src/main/java/com/moh/mhealth/pages/MainActivity.java
package com.moh.mhealth.pages;

import androidx.appcompat.app.AppCompatActivity;
import androidx.room.Room;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import com.moh.mhealth.Global_Helper;
import com.moh.mhealth.PatientDatabase;
import com.moh.mhealth.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        EditText username, password;
        username = (EditText) findViewById(R.id.main_username);
        password = (EditText) findViewById(R.id.main_password);

        Button loginBtn = (Button) findViewById(R.id.main_login_button);

        loginBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                String user = username.getText().toString();
                String pass = password.getText().toString();
                
                if (user.equals(Global_Helper.USERNAME) && pass.equals(Global_Helper.PASSWORD)) {
                    Toast.makeText(MainActivity.this, "Welcome", Toast.LENGTH_SHORT).show();

                    Intent intent = new Intent(getApplicationContext(), PatientList.class);
                    startActivity(intent);
                } else {
                    Toast.makeText(MainActivity.this, "Invalid Credentials", Toast.LENGTH_SHORT).show();
                }
            }
        });
    }
}

File: app/src/main/java/com/moh/mhealth/pages/MainActivity.kt
package com.moh.mhealth.pages

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.moh.mhealth.GlobalHelper
import com.moh.mhealth.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val username: EditText = findViewById<View>(R.id.main_username) as EditText
        val password: EditText = findViewById<View>(R.id.main_password) as EditText
        val loginBtn = findViewById<View>(R.id.main_login_button) as Button
        loginBtn.setOnClickListener {
            val user = username.text.toString()
            val pass = password.text.toString()
            if (user == GlobalHelper.USERNAME && pass == GlobalHelper.PASSWORD) {
                Toast.makeText(this@MainActivity, "Welcome", Toast.LENGTH_SHORT).show()
                val intent = Intent(applicationContext, PatientList::class.java)
                startActivity(intent)
            } else {
                Toast.makeText(this@MainActivity, "Invalid Credentials", Toast.LENGTH_SHORT).show()
            }
        }
    }
}